examples/randomized-benchmark/two-qubit-rb/readme.md

```diff
@@ -0,0 +1,86 @@
+# Randomized Benchmarking for two-qubit gates using baking
+
+Randomized benchmarking (RB) is a procedure to generate an 
+average figure-of-merit characterizing the fidelity of 
+qubit operations. This procedure is useful as full process tomography 
+may be prohibitively costly in terms of the number of required operations.   
+RB utilizes operations from the Clifford group, which in the case of two qubits contains 11520 elements
+(for $$n$$ qubits $$
+\left|\mathcal{C}_{n}\right|=2^{n^{2}+2 n} \prod_{j=1}^{n}\left(4^{j}-1\right)
+$$).
+
+We propose to illustrate the use of baking tool to perform this protocol.
+
+## The class *RBTwoQubits*
+
+The pattern behind RB is always the same: 
+- Generate a random sequence composed of Cliffords
+- Append to the sequence one Clifford reverting the action of the random sequence, such that the identity circuit is played
+- Measure the survival probability
+- Repeat prior steps for varying number of Cliffords within the sequence
+
+In order to have an easy way to derive all Cliffords of the two qubit Clifford group, 
+we use the decomposition illustrated in the following reference: https://arxiv.org/pdf/1210.7011.pdf.
+This decomposition allows an easy derivation of the group from the Clifford group for one qubit, and uses three main two qubit gates
+(that shall be decomposed later into a native set of gates by the user depending on its own hardware):
+- CNOT
+- SWAP
+- iSWAP
+
```

Comment:
what about CPHASE?

Reply:
CPHASE is irrelevant, as it is not a gate that allows the selection of a Clifford according to its class type. In the example, we assume that CPHASE is the native gate, but the user shall put a macro for his own native two qubit gate (but not mandatory) and from there derive sequences of instructions in those three indicated baking macros to yield the intended gates that do create the class separation